cmd/vic-machine/configure/configure.go

```diff
@@ -221,6 +231,14 @@ func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n
 	if n.RegistryCertificateAuthorities != nil {
 		o.RegistryCertificateAuthorities = n.RegistryCertificateAuthorities
 	}
+
+	if n.InsecureRegistries != nil {
```

Comment:
OK I'm going to skip copying this over for now and focus on doing one thing at a time.